ufc/tests/test-case-boolean-one-of-matches.json

```diff
@@ -67,6 +67,21 @@
         "subjectKey": "mike",
         "subjectAttributes": {"not_one_of_flag": "false"},
         "assignment": 0
+      },
+      {
+        "subjectKey": "nicole",
+        "subjectAttributes": {"null_flag": "null"},
+        "assignment": 5
```

Comment:
to make sure I am understanding - you want `null` to be a string `"null"` that's why nicole has an non-zero assignment

Reply:
I think what's happening for nicole is `["null"].contains("null")` => true

Reply:
No -- Nicole has the attribute set to the string `"null"`, which matches the rule and hence Nicole gets assignment 5. Owen has value `null`, which always evaluates to false, so Owen does get a zero-assignment